public/dorequest.php

```diff
@@ -232,11 +233,20 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
         $response['Response'] = getGamesListDataNamesOnly($consoleID);
         break;
 
-    case "officialgameslist":
+    case "officialgameslist": // TODO: is this used anymore? It's not used by the DLL.
         $consoleID = (int) request()->input('c', 0);
         $response['Response'] = getGamesListDataNamesOnly($consoleID, true);
         break;
 
+    case "gameinfolist":
+        $gamesCSV = request()->input('g', '');
+        if (empty($gamesCSV)) {
+            return DoRequestError("You must specify which games to retrieve info for", 400);
+        }
+        $response['Response'] = Game::whereIn('ID', explode(',', $gamesCSV))
+            ->select('Title', 'ID', 'ImageIcon')->get()->toArray();
+        break;
```

Comment:
Should we consider adding an upper bound to the count of game IDs allowed in the call?

Even with nginx's default maximum URL length of 8,192 characters, someone could craft a GET request which fetches data for ~1,500 games per request.

Reply:
The intended client (the DLL) won't be requesting any more than 20. I'm fine with adding an upper limit. The public APIs seem to use 100 for similar requests